src/services/email/serverless.yml

```diff
@@ -169,3 +216,70 @@ resources:
             - "subscription"
           SnsDestination:
             TopicARN: !Ref EmailEventTopic
+
+    SinkEmailTrigger:
```

Comment:
Sounds great - I admit, I copied the sink yml from the data service without really knowing what parts I needed and, since it worked... well, I missed that it is redundant/ not a great idea to have master's topic influencing emails from other branches - thanks!

Thinking about it though, I bet I would've figured this out very quickly after the branch got merged to Master... since right now it is My email address receiving all the emails :) Saved me from a flood :)